packages/components/organisms/f-registration/README.md

```diff
@@ -7,80 +7,115 @@
 Registration Component.
 
 </div>
+
 ---
 
 [![npm version](https://badge.fury.io/js/%40justeat%2Ff-registration.svg)](https://badge.fury.io/js/%40justeat%2Ff-registration)
 [![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
 [![Coverage Status](https://coveralls.io/repos/github/justeat/f-registration/badge.svg)](https://coveralls.io/github/justeat/f-registration)
 [![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-registration/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-registration?targetFile=package.json)
 
+---
 
 ## Usage
 
-1.  Install the module using NPM or Yarn:
+### Installation
 
-    ```bash
-    yarn add @justeat/f-registration
-    ```
+Install the module using NPM or Yarn:
+
+```sh
+yarn add @justeat/f-registration
+```
 
-    ```bash
-    npm install @justeat/f-registration
-    ```
+```sh
+npm install @justeat/f-registration
+```
 
-2.  Import the component
+### Vue Applications
 
-    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+You can import it in your Vue SFC like this (please note that styles have to be imported separately):
 
-    ```
-    import RegistrationComponent from '@justeat/f-registration';
-    import '@justeat/f-registration/dist/f-registration.css';
+```js
+import RegistrationComponent from '@justeat/f-registration';
+import '@justeat/f-registration/dist/f-registration.css';
 
-    export default {
-        components: {
-            RegistrationComponent
-        }
+export default {
+    components: {
+        RegistrationComponent
     }
-    ```
+}
+```
 
-    If you are using Webpack, you can import the component dynamically to separate the header bundle from the main `bundle.client.js`:
+If you are using Webpack, you can import the component dynamically to separate the `mega-modal` bundle from the main `bundle.client.js`:
 
-    ```
-    import '@justeat/f-registration/dist/f-registration.css';
+```js
+import '@justeat/f-registration/dist/f-registration.css';
 
-    export default {
-        components: {
-            ...
-            RegistrationComponent: () => import(/* webpackChunkName: "vue-registration" */ '@justeat/f-registration')
-        }
+export default {
+    components: {
+        // …
+        RegistrationComponent: () => import(/* webpackChunkName: "vue-registration" */ '@justeat/f-registration')
     }
+}
+```
+
+## Configuration
+
+### Props
+
+There may be props that allow you to customise its functionality.
+
+The props that can be defined are as follows (if any):
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+
+### CSS Classes
+
+The modal has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.
+
+In addition to this, the modal exposes some classes which you can target in your application.
```

Comment:
```suggestion
Registration has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.

In addition to this, registration exposes some classes which you can target in your application.
```